arrows/ceres/types.h

```diff
@@ -61,37 +44,36 @@ StringToLossFunctionType(std::string value, LossFunctionType* type);
 KWIVER_ALGO_CERES_EXPORT ::ceres::LossFunction*
 LossFunctionFactory(LossFunctionType type, double scale=1.0);
 
-/// Provide a string representation for a LensDisortionType value
-KWIVER_ALGO_CERES_EXPORT const char*
-LensDistortionTypeToString(LensDistortionType type);
-
-/// Parse a LensDistortionType value from a string or return false
-KWIVER_ALGO_CERES_EXPORT bool
-StringToLensDistortionType(std::string value, LensDistortionType* type);
-
-/// Return the number of distortion parameters required for each type
-KWIVER_ALGO_CERES_EXPORT unsigned int
-num_distortion_params(LensDistortionType type);
+/// The options for camera intrinsic sharing supported in the config
```

Comment:
```suggestion
/// The options for camera intrinsic sharing supported in the config.
```
(Looks like various addition places in this file.)